docs/source/data/fetching-rest.mdx

```diff
@@ -289,13 +358,13 @@ When layering GraphQL over REST APIs, it's most helpful to have a resource cache
 - Can be shared across multiple GraphQL servers
 - Provides cache management features like expiry and invalidation that use standard HTTP cache control headers
 
-#### Batching with REST APIs
+### Batching with REST APIs
 
 Most REST APIs don't support batching. When they do, using a batched endpoint can _jeopardize_ caching. When you fetch data in a batch request, the response you receive is for the exact combination of resources you're requesting. Unless you request that same combination again, future requests for the same resource won't be served from cache.
 
 We recommend that you restrict batching to requests that _can't_ be cached. In these cases, you can take advantage of DataLoader as a private implementation detail inside your `RESTDataSource`:
 
-```ts 
+```ts
 class PersonalizationAPI extends RESTDataSource {
   constructor() {
```

Comment:
We can update this to what we've been doing (remove constructor, and `override baseURL = ...`) in the class body

Reply:
Pretty sure `willSendRequest` below needs an `override`